package.json

```diff
@@ -35,9 +36,11 @@
     "throat": "^5.0.0"
   },
   "devDependencies": {
-    "@babel/cli": "^7.0.0",
-    "@babel/core": "^7.0.0",
-    "@babel/preset-env": "^7.0.0",
+    "@babel/cli": "^7.12.1",
```

Comment:
we don't need to bump the range here, just `rm yarn.lock && yarn` in root should be fine. Can you open that up as a separate PR?

Reply:
#42 , I'll revert the versions of `@babel/*` to `^7.0.0` after it has been merged